### ISSUE TITLE
[Windows-vlc] Scrubbing stops episodes entirely

### ISSUE THREAD

**Issue by A0A3R2E6** [pystardust/ani-cli#82]
While watching any episode, with a stable quality and connection, letting episodes play from start to finish is no problem at all, but scrubbing back or forward stops any episode from playing again until VLC is closed and an episode is selected again, or at least, the amount of time on which the episode could start playing again is way inconvenient and not acceptable at all.

**Comment by ura43** [comment]
Did you find a way to scrub without trouble?

**Comment by A0A3R2E6** [comment]
Sadly, no.

**Comment by ura43** [comment]
You are running the latest version?
Is the issue just scrubbing forward or is scrubbing back also not without troubles?
Do you have an even moderately strong computer? 
Because if you have garbage tier hardware that might be the unfortunate cause

**Comment by A0A3R2E6** [comment]
I'll have to go back to you on that one later since there seems to be another problem with ani-cli currently where it says "No search results found". But to address the points in your comment:

.- Yes, I've updated to the latest version, encountered the problem I said above, uninstalled and installed again following the Master branch instructions.

.- Yes, the issue is scrubbing both forward and backwards.

.- I believe the only subpar component in my current PC is my processor, since I'm using an Athlon 3000g. I haven't tried ani-cli on Linux yet, but I suspect my processor wouldn't be the deciding factor on scrubbing.

I've tried streaming with VLC before, and this kind of behavior has only happened with ani-cli, although I'll have to try again after the "No search results found" is fixed. It may have to do with gogoanime, rather than with ani-cli itself.
